src/client/client4.ts

```diff
@@ -3494,6 +3494,8 @@ export default class Client4 {
                 search: '',
                 title: '',
                 url: '',
```

Comment:
@Adovenmuehle can you confirm these other props are ok for `page` events?

Reply:
Yes this looks correct to me.